src/main/java/org/gridsuite/securityanalysis/server/repositories/CommonLimitViolationRepository.java

```diff
@@ -28,7 +29,8 @@ public interface CommonLimitViolationRepository<T> {
      */
     default Specification<T> getParentsSpecifications(
         UUID resultUuid,
-        List<ResourceFilterDTO> filters
+        List<ResourceFilterDTO> filters,
+        boolean isSubjectLimitViolations
```

Comment:
it could be nice to use specific repository implementations instead of a boolean to achieve this 

maybe create a method like "addSpecificFilter" in CommonLimitViolationRepository which would be implemented differently in SubjectLimitViolationRepository and in ContingencyRepository

Reply:
I agree with that 
done